compiler/syntax/src/res_io.ml

```diff
@@ -11,4 +11,4 @@ let write_file ~filename ~contents:txt =
   let chan = open_out_bin filename in
   output_string chan txt;
   close_out chan
-[@@raises Sys_error]
+[@@raises Sys_error] [@@dead "+write_file"]
```

Comment:
interesting

Reply:
Yes, don't know why this is necessary now.